docroot/themes/humsci/humsci_basic/templates/views/pattern-view-field.html.twig

```diff
@@ -35,8 +35,27 @@
 {{ attach_library(default_theme ~ '/pattern-view-field') }}
 
 {%- set has_content = field.content|render|striptags('<span><div>')|trim -%}
+{%- set field_element_is_heading = field.element_type != 'div' and field.element_type != 'span' and field.element_type != '' -%}
+
+{# Condition to check for custom, course or person class in field wrapper #}
+{%- set field_wrapper_has_hb_class = false -%}
+{%- for class in field.wrapper_attributes.class -%}
+  {%- if not field_wrapper_has_hb_class and (class starts with 'hb' or 'course' in class or 'person' in class)-%}
```

Comment:
We're assuming a lot here. There are hundreds of views in the platform and we're adding very specific logic that might not match all cases. Let's better leave views untouched.